pkg/metaclient/op.go

```diff
@@ -0,0 +1,310 @@
+// Copyright 2016 The etcd Authors
```

Comment:
If this file is modified based on a third-party work, could you add a link to the original? Also please read the Redistribution section in https://www.apache.org/licenses/LICENSE-2.0

Reply:
I have already added the apache license 2.0 file to the project and the license head for op.go file.  PTAL
Should I add a link to the original? I don't find any instruction on it.

Reply:
I think a link to the original may help the maintenance of the code.

Reply:
Done. PTAL